liquibase-core/src/main/java/liquibase/configuration/LiquibaseConfiguration.java

```diff
@@ -158,7 +158,14 @@ public <DataType> ConfiguredValue<DataType> getCurrentConfiguredValue(Configurat
                     if (foundFirstValue) {
                         logMessage.append("Overrides ");
                     }
-                    logMessage.append(StringUtil.lowerCaseFirst(providedValue.describe()));
+
+                    String describe = providedValue.describe();
+                    char[] chars = describe.toCharArray();
+                    if (chars.length >= 2 && Character.isUpperCase(chars[0]) && Character.isUpperCase(chars[1])) {
+                        logMessage.append(describe);
+                    } else {
+                        logMessage.append(StringUtil.lowerCaseFirst(describe));
+                    }
```

Comment:
Can you put a comment in the code explaining the purpose of this? I can see this causing some head-scratching in the future when we forget why we did this.